xml/System.Buffers/SequenceReaderExtensions.xml

```diff
@@ -40,8 +40,8 @@
       <Docs>
         <param name="reader">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads a big-endian <see cref="T:System.Int16" />.</summary>
```

Comment:
There's something wrong with this file. It doesn't contain all 6 of the relevant APIs, but rather just 3:
https://github.com/dotnet/corefx/blob/97d5957fc71853961082e22873d926269b291b92/src/System.Memory/ref/System.Memory.cs#L396-L404

Is something broken in how the reflection tool is generating this?

Also note, there is Int16, Int64 mismatch happening in the API signature vs the description:
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.sequencereaderextensions?view=netcore-3.0
![image](https://user-images.githubusercontent.com/6527137/61913887-6cf82b00-aef3-11e9-94ea-2eebf68d3ae7.png)

It is correctly documented in source:
https://github.com/dotnet/corefx/blob/97d5957fc71853961082e22873d926269b291b92/src/System.Memory/src/System/Buffers/SequenceReaderExtensions.Binary.cs#L136-L154